audit-service/README.md

```diff
@@ -38,11 +45,21 @@ been implemented. This service does not have any output Kafka topics.
 
 *The token value come from the headers of the Kafka message that the service receives. This links the audit message to the original request that was made.
 
-If an error has occurred during a request/read then an AuditErrorMessage will be consumed from the `error` Kafka topic. This type of message can be sent from any of the Palisade services
-(e.g user-service or policy-service).
+If an error has occurred at any stage during either the request or read phases then an AuditErrorMessage will be added to the `error` Kafka topic by the service that encountered the issue. 
+This type of message can be sent from any of the Palisade services (e.g. User Service or Policy Service).
+This message will then be read by the Audit Service and passed onto the local Audit Service implementation to allow the details of the error to be logged.
```

Comment:
This section needs more detail to explain what it looks like how it works.  The reader will be a developer who wants to know how they can implement this.  If we have an example or a test it would be helpful to point it out to explain.  Same goes for the back-up for messages that could not be deserialised.

Reply:
I have added information about the interface at the start of the ReadMe now. This section has also been re-worded.